tests/ser.rs

```diff
@@ -335,9 +333,7 @@ fn vec_of_bool_to_non_empty_slice() {
     }
     assert_eq!(
         buffer,
-        &[
-            12, 255, 129, 2, 1, 2, 255, 130, 0, 1, 2, 0, 0, 6, 255, 130, 0, 2, 1, 0
-        ]
+        &[12, 255, 129, 2, 1, 2, 255, 130, 0, 1, 2, 0, 0, 6, 255, 130, 0, 2, 1, 0,]
```

Comment:
`rustfmt` did this, let me know if you want me to fix :-/

Reply:
Hmm, what version of `rustfmt` are you using? `cargo fmt --version` gives me 
`0.3.8-nightly (346238f 2018-02-04)`.

Reply:
`rustfmt 0.6.0-nightly (ac8ae006 2018-04-25)`

I'll just revert the formatting changes for now.